elasticsearch/mapping_sirene_index.py

```diff
@@ -267,6 +269,7 @@ class ElasticsearchSireneIndex(Document):
     nom_raison_sociale = Text(analyzer=annuaire_analyzer, fields={"keyword": Keyword()})
     nombre_etablissements = Integer()  # NaN can't be stored in an integer array
     nombre_etablissements_ouverts = Integer()
+    note_egapro = Float()
```

Comment:
I would avoid adding note or at least move it to `details_egapro : [{
	annee,
	note
}]` at siren level